config/jobs/kubernetes/test-infra/test-infra-canaries.yaml

```diff
@@ -1,28 +1,23 @@
-presubmits:  
+presubmits:
   kubernetes/test-infra:
   - name: pull-test-infra-bazel-canary
-    always_run: false
+    skip_report: false
+    always_run: true
```

Comment:
We shouldn't make the canary job blocking.

Reply:
Done